jgrapht-core/src/test/java/org/jgrapht/alg/decomposition/ChordalityNiceTreeDecompositionTest.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * (C) Copyright 2018-2018, by Ira Justus Fesefeldt and Contributors.
+ *
+ * JGraphT : a free Java graph-theory library
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation, or (at your option) any
+ * later version.
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+package org.jgrapht.alg.decomposition;
+
+
+import static org.junit.Assert.assertTrue;
+
+import org.jgrapht.*;
+import org.jgrapht.alg.cycle.ChordalityInspector;
+import org.jgrapht.generate.*;
+import org.jgrapht.graph.*;
+import org.jgrapht.graph.builder.*;
+import org.jgrapht.util.SupplierUtil;
+import org.junit.*;
+
+/**
+ * Tests for the {@link ChordalNiceTreeDecomposition}
+ *
+ * @author Ira Justus Fesefeldt (PhoenixIra)
+ */
+public class ChordalityNiceTreeDecompositionTest
```

Comment:
Since the class that is being tested is `ChordalNiceTreeDecomposition`, this one should be named `ChordalNiceTreeDecompositionTest`